etc/kayobe/ansible/rekey-hosts.yml

```diff
@@ -0,0 +1,97 @@
+---
+# Playbook to rotate SSH keys across the cloud. By default it will rotate the
+# standard keys used by kayobe/kolla-ansible, but it can be configured for any
+# keys.
+
+- name: Rekey hosts
+  hosts: overcloud,seed,seed-hypervisor,infra-vms
+  gather_facts: false
+  vars:
+    ansible_ssh_common_args: "-o StrictHostKeyChecking=no"
```

Comment:
Why do we need this?

Reply:
I'll retest this bit soon. Unsure it it was required before I hadn't connected to the other hosts before or it legitimately needs it because the key is changing